debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.3.107-3) unstable; urgency=medium
+  [ Lei Huang ]
+  * Add libcapstone as a new prerequisite package
```

Comment:
You should also add a comment about the new file being installed in the -client RPM here.

Reply:
Put "Add libpil4dfs.so in daos-client rpm" as you suggested. Thank you!